bedrock/legal/templates/legal/terms/firefox-relay.html

```diff
@@ -0,0 +1,7 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "legal/docs-base.html" %}
+
+{% block page_title %}{{ _('Firefox Relay: About Your Rights') }}{% endblock %}
```

Comment:
This is a new string right? Should we use Fluent instead?

Reply:
These legal pages all appear to use this type of title string that has never been localised. We should figure out what to do about that, and it should definitely use Fluent if a string is needed. For now though, I think this is probably OK as it sticks with the existing convention?

Reply:
Updated to a hard-coded English string for now.

Reply:
Yeah. Good point. I'm wondering if we couldn't just yank the title from the document itself for use in the title, but we should save that for a larger rework of all of these. Hardcoded or just leaving it like the others is fine with me 👍

Reply:
Yeah let’s tackle this when we port the legal pages to Fluent. Thanks @pmac !